drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/handlers/NewDecisionTableXLSHandlerTest.java

```diff
@@ -85,13 +90,20 @@
 
     @Before
     public void setup() {
+        newResourceSuccessEventMock = spy( new EventSourceMock<NewResourceSuccessEvent>() {
```

Comment:
You might be better just ```@Mock EventSourceMock<NewResourceSuccessEvent> newResourceSuccessEventMock;``` (I only comment as I also used to use a concrete instances of ```EventSourceMock``` and override the ```fire``` method - mocking it seemed to provide the exact same function without the boiler plate).